apps/etl-func/src/adapters/pdv-tokenizer/pii-resource.ts

```diff
@@ -0,0 +1,4 @@
+import { z } from "zod";
+
+export const piiResourceSchema = z.object({ pii: z.string().min(1) });
```

Comment:
Why do we need this to be an object? Can't we just use a string?
```suggestion
export const piiResourceSchema = z.string().min(1);
```

Reply:
this is the request of the pdv tokenizer api and it must be an object with a field named pii that is a string. You can check the openapi [here](https://api.tokenizer.pdv.pagopa.it/docs/tokenizerpapis/openapi.json)